### PR TITLE
[bitnami/mysql] Support templating in initdbScriptsConfigMap

### PR DESCRIPTION
**Pull request by yisiqi**
Signed-off-by: Yi Siqi <yisq@me.com>

**Description of the change**

Support overriding mysql init db configmap by parent chart.

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

Similar to #9612

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/mysql/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,8 @@ name: mysql
 sources:
   - https://github.com/bitnami/bitnami-docker-mysql
   - https://mysql.com
+<<<<<<< support-overriding-mysql-init-db-configmap
+version: 8.8.29
+=======
 version: 8.8.30
+>>>>>>> master
```
Comment (rafariossaa) [inline bitnami/mysql/Chart.yaml:32]: Hi,
  Could you take a look to this ? I think you had some changes pending .

**Review by rafariossaa — CHANGES_REQUESTED** [review]

File: bitnami/mysql/Chart.yaml (hunk 1 above)
Comment (yisiqi) [reply bitnami/mysql/Chart.yaml]: Sorry, I fixed it.

**Comment by yisiqi** [comment]
@rafariossaa Please help to review.

**Review by rafariossaa — APPROVED** [review]
Thanks !

LGTM

**Pull request merged**